.gdbinit-kftrace

```diff
@@ -0,0 +1,31 @@
+# some generic settings
+set output-radix 0x10
+set pagination off
+set confirm off
+set verbose off
+
+# Command to easily append KFT events to specified file
+define dump-kft
+  printf "Appending %d entries to file `dump.kft`\n", kft_used
+  append memory dump.kft kft_event_list kft_event_list+kft_used
+end
+
+set $kft_flush_count = 0
+
+break kft_flush
```

Comment:
If `kft_event_list` was always compiled into the kernel you could use conditional breakpoint here, i.e. `break kft_flush if kft_event_list`. Perhaps then it could be integrated with regulard `.gdbinit` file?

Reply:
changed. But I think it is better to keep this there to make the launch with KFT a separate target.